shims/spark33/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFileFormat.scala

```diff
@@ -65,10 +65,8 @@ class OrcFileFormat extends FileFormat with DataSourceRegister with Serializable
       sparkSession: SparkSession,
       options: Map[String, String],
       files: Seq[FileStatus]): Option[StructType] = {
-    if (
-      "true".equals(sparkSession.sparkContext.getLocalProperty("isNativeAppliable"))
-      && GlutenConfig.isCurrentBackendVelox && false
-    ) {
+    // why if (false)? Such code requires comments when being written
```

Comment:
We'd better follow the same comment style by capitalizing the first letter and add `.` at the end for all comments.

Reply:
Sure.